examples/vortex.py

```diff
@@ -0,0 +1,139 @@
+__copyright__ = """
```

Comment:
Not surprising.  The template I work from when adding examples, has the same:
https://github.com/inducer/grudge/blob/master/examples/wave/wave-eager.py
https://github.com/inducer/grudge/blob/master/examples/wave/wave-eager-mpi.py

I assumed this was so that even if the user doesn't have MPI they can still run the examples?

Reply:
Both are also useful.  CI, and TEESD run both the serial and the parallel versions.  On some compute platform - we could catch an issue with the MPI install or the mirgecom build (wrt MPI).